GameFighter/Models/Lancer.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace GameFighter.Models
+{
+    public class Lancer : Warrior
+    {
+        public Lancer()
+        {
+            Attack = 6;
+        }
+
+        public override void Attacks(Warrior warrior, Warrior warriorBehind)
```

Comment:
This solution isn't extensible. What if we need to add damage to more than 2 warriors? Are you going to add one more parameter?